src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  justify-content: space-between;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+
+  &__image {
+    align-self: center;
+
+    width: 160px;
+    height: 134px;
+    padding-bottom: 40px;
+  }
+
+  &__title {
+    padding-bottom: 4px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: var(--main-accent);
+  }
+
+  &__code {
+    padding-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: var(--secondary);
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 16px;
+  }
+
+  &__price-title {
+    color: var(--secondary);
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    color: var(--main-accent);
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    border: 1px transparent;
+    background: var(--blue-accent);
+
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+  }
+
+  &:hover > .card__button {
+    border: 1px solid var(--blue-accent);
```

Comment:
9 [CHECKLIST] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover